### DISCUSSION TITLE
Custom Device Manager as width/height inputs

### DISCUSSION THREAD

**Discussion by badiuciprian** [GrapesJS/grapesjs#4472] [Q&A]
Hy guys, really excited about grapesjs, but I have a question:
Is it possible to have the device manager as 2 inputs, one for page width and one for page height?
I want to be allow the user to set specific page size from the ui.

**Comment by artf** [comment]
You should be able to accomplish that by updating the device properties. Currently, the iframe size is not reacting directly to the device properties changes but you should be able to trigger that manually (probably something I can fix for the next release).

```js
// Get the selected device or use editor.Devices.get('device-id')
const device = editor.Devices.getSelected();
// Update the device
device.set({ width: '500px', height: '...',  widthMedia: '...' });
// Trigger manually the iframe update
editor.Canvas.getModel().updateDevice();
```

**Reply by badiuciprian** [reply]
thank you!
